pkg/actors/targets/workflow/workflow.go

```diff
@@ -208,6 +209,10 @@ func (w *workflow) InvokeMethod(ctx context.Context, req *internalsv1pb.Internal
 func (w *workflow) executeMethod(ctx context.Context, methodName string, request []byte) ([]byte, error) {
 	log.Debugf("Workflow actor '%s': invoking method '%s'", w.actorID, methodName)
 
+	if w.actorState == nil {
+		return nil, messages.ErrActorRuntimeNotFound
```

Comment:
nit: should this err be more verbose to the user clearly stating that this needs to be enabled and point to docs?

Reply:
I don't know how we can be any more verbose- this error is piped and returned to the client API call as a failure.